src/elements/common/_animations.scss

```diff
@@ -0,0 +1,27 @@
+@keyframes fade-in {
```

Comment:
This seems likely to conflict with other mixins. Should it be namespaced with `bdl`?